packages/openapi3/src/openapi.ts

```diff
@@ -1613,7 +1613,7 @@ function createOAPIEmitter(program: Program, options: ResolvedOpenAPI3EmitterOpt
     const isString = isStringType(program, getPropertyType(typespecType));
     const isNumeric = isNumericType(program, getPropertyType(typespecType));
 
-    if (!target.description && docStr) {
```

Comment:
Do you know what the reason was for this check previously?

Reply:
My guess it was something like applying the decorators for a property vs directly on model/scalar. And we didn't want the decorator on the model/scalar to override the ones on the property.

I was kinda expecting other tests to fail but seems like this might have already been fixed. 

I guess I should also update the other check to be consitent.

Reply:
Test still seems to pass, and there is not change to the samples(`typespec-azure` pr one has some but those are expected)

Think this was most likely change either when `scalar` were added or when trying to fix some other issue.

Reply:
https://cadlplayground.z22.web.core.windows.net/prs/1918/?c=bW9kZWwgRm9vIHsKICBAZG9jKCJQcm9wIGRvYyIpCiAgcHJvcFdpdGhEb2M6IGN1c3RvbVN0cjsKCiAgc2NhbGFyT25sedYaTGV2ZWwyOiBzdXBlcsshfQoKxmlDxRQgc3RyaW5nIikKxjfKTCBleHRlbmRzxyI7Csg5T3ZlcnJpZGUg10LOctFH

Reply:
Looks like there was a sample output change and git check for changed files is failing. Probably a desirable change.

Reply:
It is an expected change for what the spec was saying but I think `url` scalar shouldn't have `@format("url")`

Reply:
or shouldn't it and should I update the sample

Reply:
What was the diff? Asking to see if I have an opinion on which solution is better. :)

Reply:
It was setting `format: url` instead of `format: uri` for the `url` type.

Reply:
but we did have logic that uri and url scalar should do format: uri in the openapi emitter. So the format decorator was overriding that.

Reply:
I'll keep it this way because that's the non-breaking behavior, we can discuss a change if someone feels like it